source/reference/operator/aggregation/isNumber.txt

```diff
@@ -0,0 +1,193 @@
+=======================
+$isNumber (aggregation)
+=======================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+Definition
+----------
+
+.. expression:: $isNumber
+
+   .. versionadded:: 4.4
+
+   :expression:`$isNumber` checks if the specified 
+   :ref:`expression <aggregation-expressions>` resolves to one of the
+   following numeric :term:`BSON types`:
+
+   - :bsontype:`Integer <Int32>`
+   - :bsontype:`Decimal <Decimal128>`
+   - :bsontype:`Double <Double>`
+   - :bsontype:`Long <Int64>`
+
+   :expression:`$isNumber` returns:
+
+   - ``true`` if the expression resolves to a number.
+
+   - ``false`` if the expression resolves to any other
+     :doc:`BSON type </reference/mongodb-extended-json>`, ``null``, or
+     an empty ``{}`` result. 
```

Comment:
So ... by empty ``{}`` result -- are you stating that it's an empty document? so bson type object?

empty (as in missing) is not ``{}``  -- if you meant empty/missing instead of an empty document, we should update.

Reply:
Fixed as discussed.